llvm/lib/CodeGen/AsmPrinter/AsmPrinter.cpp

```diff
@@ -868,14 +881,27 @@ void AsmPrinter::emitGlobalVariable(const GlobalVariable *GV) {
 
   MCSymbol *EmittedInitSym = GVSym;
 
+  if (GV->hasAttribute(Attribute::SanitizeAddress)) {
```

Comment:
This would be replaced by "asan_instrumented" attribute, once #68865 gets accepted.